src/NoTimeToWaitBundle/Resources/views/Default/nttw4.html.twig

```diff
@@ -58,7 +58,21 @@ Closing: 17:00</p>
 
 <h2>Program</h2>
 
-<p>(To be announced) A Call for Proposals shall be issued soon.</p>
+<p>We are now accepting proposals. Please submit your ideas for presentations, lightning talks or panels at https://bit.ly/nttw4proposals (where you can see ideas already proposed) or submit ideas for the program <a href="{{ path('ma_contact') }}"> by emailing the conference organizers directly</a>. We’d like to encourage presentations from a variety of perspectives related to our themes of:</p>
```

Comment:
Link is not automatically created by our tools, you need to manually write it ([name](url) method is fine)

Reply:
i revised the links